frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -17,12 +18,25 @@ import {
   PostFormSubmissionAction,
   CustomFeatureFlag,
 } from '@console/plugin-sdk';
+import { ALLOW_SERVICE_BINDING_FLAG } from '@console/topology/src/const';
+import { TopologyDataModelFactory } from '@console/topology/src/extensions';
 import { doConnectsToBinding } from '@console/topology/src/utils/connector-utils';
 import { getGuidedTour } from './components/guided-tour';
+import { getBindableServiceResources } from './components/topology';
```

Comment:
your vendor size is increased to 3.91 MB. import this using absolute path
```suggestion
import { getBindableServiceResources } from './components/topology/bindable-service-resources';
```